conf/application.conf

```diff
@@ -53,6 +53,20 @@ microservice {
       host = localhost
       port = 9402
     }
+    des {
+      host=localhost
+      port=9904 #agent-stubs
```

Comment:
pedantic: agents-external-stubs expected to be listening in on 9904

Reply:
i think this actually is the agent-stubs port and then agents-external-stubs intercepts it...?

Reply:
Ah yeah, you're right